VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -14,9 +14,24 @@
 
 cfg.Analyzer.nosubdir = True
 
+#Create shifted MET Ntuples
+shifted_met_keys = ["met_shifted_{0}".format(n) for n in range(14)]
+shifted_mets = {mk: NTupleObject(mk, metType, help="PF E_{T}^{miss}, after default type 1 corrections, shifted with %s" %mk) for mk in shifted_met_keys}
+
+#Need to define global objects here to add shifted MET
+globalObjects = {
+          "met"    : NTupleObject("met",     metType, help="PF E_{T}^{miss}, after default type 1 corrections"),
+          "fakeMET"    : NTupleObject("fakeMET", fourVectorType, help="fake MET in Zmumu event obtained removing the muons"),
+          "H"    : NTupleObject("H", fourVectorType, help="higgs"),
+          "HCSV"    : NTupleObject("HCSV", fourVectorType, help="higgs CSV selection"),
+                "H3cj"    : NTupleObject("H3cj", fourVectorType, help="higgs 3 cen jets selection"),
+          "V"    : NTupleObject("V", fourVectorType, help="z or w"),
+}
+globalObjects.update(shifted_mets)
+
```

Comment:
why? can't you do simply 
treeProducer.globalObjects.update(shifted_mets)
after the initial declaration? I'd say so